todo/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.core.exceptions import ValidationError
+
+from todo.models import Tag, Task
+
+
+class TagForm(forms.ModelForm):
+    class Meta:
+        model = Tag
+        fields = "__all__"
+
+
+class TaskForm(forms.ModelForm):
+    tags = forms.ModelMultipleChoiceField(
+        queryset=Tag.objects.all(),
+        widget=forms.CheckboxSelectMultiple,
+        required=False
+    )
```

Comment:
We can also add a widget for the `deadline` field here to improve the user experience when entering a deadline in a datetime format:

```suggestion
class TaskForm(forms.ModelForm):
    tags = forms.ModelMultipleChoiceField(
        queryset=Tag.objects.all(),
        widget=forms.CheckboxSelectMultiple,
        required=False
    )
    deadline = forms.DateTimeField(
        widget=forms.DateTimeInput(
            attrs={"type": "datetime-local"}
        ),
        required=False
    )
```